tests/BrowsershotTest.php

```diff
@@ -301,4 +301,25 @@ public function it_can_run_without_sandbox()
             ],
         ], $command);
     }
+
+    /** @test */
+    public function it_can_ignore_https_errors()
+    {
+        $command = Browsershot::url('https://example.com')
+            ->ignoreHttpsErrors()
+            ->createScreenshotCommand('screenshot.png');
+
+        $this->assertEquals([
+            'url' => 'https://example.com',
+            'action' => 'screenshot',
+            'options' => [
+                'ignoreHttpsErrors' => true,
+                'path' => 'screenshot.png',
+                'viewport' => [
+                    'width' => 800,
+                    'height' => 600,
+                ],
+            ],
+        ], $command);
+    }
```

Comment:
I don't have much testing experience, but I'd like to get better. If this isn't what you have in mind for a test, please let me know.